<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminOrderDialog</class>
 <widget class="QDialog" name="adminOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>144</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>房间预订</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>301</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>66</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>客    户 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>66</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>房    间 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>66</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>预订时间 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>66</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>退房时间 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>391</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
